Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN mkdir /app
 WORKDIR /app
 ADD Gemfile /app/Gemfile
 ADD Gemfile.lock /app/Gemfile.lock
-RUN bundle install -j4
+RUN bundle install --path vendor/bundle -j4
```

Comment:
Ah, if the dependency library can be persisted in the system's Ruby it is still fine.